### DISCUSSION TITLE
Welcome to lakefs-spec Discussions!

### DISCUSSION THREAD

**Discussion by AdrianoKF** [aai-institute/lakefs-spec#44] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by caoergou** [comment]
I'd like to know if this project is still ongoing? It seems there aren't many issues or pull requests lately.

**Reply by nicholasjng** [reply]
yes, this repo is still maintained. The implementation has been stable for a while now, which is why the majority of activity is around bug fixes, and even there, we do not get many issues, so we consider things to be working fine.
